port/osx/omrosdump.c

```diff
@@ -155,6 +161,30 @@ coredump_to_file(mach_port_t task_port, pid_t pid)
 			goto done;
 		}
 		file_off += sizeof(x86_exception_state_t);
+#elif defined(OMR_ARCH_AARCH64) /* defined(OMR_ARCH_X86) */
+		written = pwrite(corefile_fd, &threads[i].thread_state, sizeof(arm_thread_state64_t), file_off);
```

Comment:
This whole section need not be AARCH64-specific if, for example, we replace
```
    pwrite(corefile_fd, &threads[i].thread_state, sizeof(arm_thread_state64_t), file_off)
```
with
```
    pwrite(corefile_fd, &threads[i].thread_state, sizeof(threads[i].thread_state), file_off)
```
To achieve this, `neon_state` should be renamed `float_state`.

Reply:
Updated as suggested.